internal/http/auth.go

```diff
@@ -213,8 +213,9 @@ func (h authHandler) validate(w http.ResponseWriter, r *http.Request) {
 		h.log.Debug().Msgf("found user session: %+v", session)
 		// Return username if available in session
 		response := map[string]interface{}{
-			"username":    session.Values["username"],
-			"auth_method": session.Values["auth_method"],
+			"username":        session.Values["username"],
+			"auth_method":     session.Values["auth_method"],
+			"profile_picture": session.Values["profile_picture"],
```

Comment:
`profile_picture` probably needs to be checked if it exists first here since it's might or might not exist on the session.